tests/components/flick_electric/test_config_flow.py

```diff
@@ -0,0 +1,104 @@
+"""Test the Flick Electric config flow."""
+import asyncio
+
+from asynctest import patch
+from pyflick.authentication import AuthException
+
+from homeassistant import config_entries, data_entry_flow, setup
+from homeassistant.components.flick_electric.const import DOMAIN
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from tests.common import MockConfigEntry
+
+CONF = {CONF_USERNAME: "test-username", CONF_PASSWORD: "test-password"}
+
+
+async def _flow_submit(hass):
+    return await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}, data=CONF,
+    )
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)